<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoAdjustDialog</class>
 <widget class="QDialog" name="VideoAdjustDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ビデオ調整</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <layout class="QGridLayout" name="_gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="3" column="0" colspan="3">
      <widget class="QSlider" name="_horizontalSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="_label">
       <property name="text">
        <string>コントラスト</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="_spinBox">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_label_2">
       <property name="text">
        <string>明るさ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="_spinBox_2">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QSlider" name="_horizontalSlider_2">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="_label_3">
       <property name="text">
        <string>鮮やかさ</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="_spinBox_3">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QSlider" name="_horizontalSlider_3">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="_label_4">
       <property name="text">
        <string>色合い</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="_spinBox_4">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QSlider" name="_horizontalSlider_4">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="_label_5">
       <property name="text">
        <string>ガンマ</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="_spinBox_5">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QSlider" name="_horizontalSlider_5">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="_buttonLoad">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>保存値へ戻す</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonSave">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_spinBox_2</tabstop>
  <tabstop>_spinBox_3</tabstop>
  <tabstop>_spinBox_4</tabstop>
  <tabstop>_spinBox_5</tabstop>
  <tabstop>_horizontalSlider</tabstop>
  <tabstop>_horizontalSlider_2</tabstop>
  <tabstop>_horizontalSlider_3</tabstop>
  <tabstop>_horizontalSlider_4</tabstop>
  <tabstop>_horizontalSlider_5</tabstop>
  <tabstop>_buttonSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>_horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
